### PR TITLE
Update /iot page

### LINKED ISSUE #6035: [COPY UPDATE] index - "Ubuntu for the Internet of Things"
Please update index - "Ubuntu for the Internet of Things" from the [copy doc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Update `/iot` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/iot
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#heading=h.c3ugpbbmbux2)

## Issue / Card

Fixes #6035

### CODE REVIEW

**Comment by solazio** [comment]
@pmahnke  can you please have a look at this [copy](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit#heading=h.c3ugpbbmbux2) as it seem to be very old...? And I don't know to what extent I should update the page.

**Comment by anthonydillon** [comment]
## QA
- The quote from "Mac Devine, IBM Vice President & CTO of Emerging Technology & Advanced Innovation" doesn't seem to be in the copy doc.
- Advantech should be removed from the logos
- Update Control section the copy is different in the copy doc
- Missing border from the bottom of the pricing include. Or remove all section borders.

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
   <div class="row u-vertically-center">
     <div class="col-6">
       <h1>Ubuntu for the Internet&nbsp;of&nbsp;Things</h1>
-      <p>From home control to drones, robots and industrial systems, Ubuntu is the <a class="p-link--inverted" href="/embedded">new standard for embedded Linux</a>. Whether you choose the classic Ubuntu Server or the new Ubuntu Core you get robust security, app stores and reliable updates.</p>
+      <p>From smart homes to smart drones, robots, and industrial systems, Ubuntu is the <a class="p-link--inverted" href="/embedded" style="font-style: italic;">new standard for embedded Linux</a>. Get the world’s best security, a custom app store, a huge developer community and reliable updates.
```
Comment (anthonydillon) [inline templates/internet-of-things/index.html]: I don't think we want to add italic links as we don't anywhere else.

Comment (solazio) [reply templates/internet-of-things/index.html]: This was @anasereijo suggestion as the link was not visible. @anasereijo can you please propose it as a vanilla pattern and/or discuss alternatives with vanilla team.

**Pull request merged**